### DISCUSSION TITLE
Custom building of admin-ui doesn't work

### DISCUSSION THREAD

**Discussion by oculos** [keycloak/keycloak#29686] [Archive]
Hi,
Just to test a bit, I built the admin-ui.
When I move to the server, and select the theme, I get this error on the console:

```
Uncaught TypeError: The specifier “react/jsx-runtime” was a bare specifier, but was not remapped to anything. Relative module specifiers must start with “./”, “../” or “/”. index-CZ2Urzjs.js:2:46
```

The admin ui doesn't load.

<img width="1486" alt="Screenshot 2024-05-20 at 08 58 51" src="https://github.com/keycloak/keycloak/assets/6791923/ef73ff2c-56d4-4f5b-9a80-9fdf71a9c424">

What could be the issue? Is there any other argument to the `mvn` command that I should have used?

**Comment by LogaMithran** [comment]
Hi there, 
I hope this thread might help you https://github.com/keycloak/keycloak/discussions/19508#discussioncomment-5619466

**Reply by oculos** [reply]
> Hi there, I hope this thread might help you [#19508 (reply in thread)](https://github.com/keycloak/keycloak/discussions/19508#discussioncomment-5619466)

Thanks, it did, indeed!
